<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChildWidgetC</class>
 <widget class="QWidget" name="ChildWidgetC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>821</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>需求名称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>科目:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>费用:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>492</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>510</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>需求说明:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_changed">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>600</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>600</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>289</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>54</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>年级:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
